cluster-bootstrap/src/test/scala/akka/management/cluster/bootstrap/LowestAddressJoinDeciderSpec.scala

```diff
@@ -78,9 +83,9 @@ class LowestAddressJoinDeciderSpec extends WordSpecLike with Matchers with Scala
         contactPointsChangedAt = now.minusSeconds(2),
         contactPoints = Set(contactA, contactB, contactC),
         seedNodesObservations = Set(new SeedNodesObservation(now.minusSeconds(1), contactA,
-            Address("akka", "sys", "127.0.0.1", 2552), Set(Address("akka", "sys", "127.0.0.1", 2552))))
+            Address("akka", "sys", "10.0.0.2", 2552), Set(Address("akka", "sys", "10.0.0.2", 2552))))
```

Comment:
Not sure if the change from 127 to 10 is important in this changeset?

Reply:
The test does not hinge on the change of IP address to `10.0.0.2`, but there's a discussion around `127.0.0.1` in #187, so I wanted to make it clear that this PR does NOT address that part of the story.

Reply:
I'm fine with keeping the change, just that it did not seem directly related to the issue at hand... We try to keep changes minimal and focused.